documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,90 @@
+# WCA Competition Requirements Policy
+<div class="version">
+### Version 1.0
+</div>
+
+## Purpose
+The purpose of this policy is to define the requirements and processes that a competition must meet and follow in order to be recognized as a WCA Competition.
+
+## Policy
+### WCA Competitions
+An official WCA Competition must:
+
+1. Be announced at least 28 days (four weeks) before the start of the competition on the WCA website.
+   1. WCA Delegates should allow a suitable time frame for competitions to be announced after they are confirmed to the WCA Competition Announcement Team.
+2. Comply with the [WCA Regulations](wca{regulations}).
+3. Comply with all [WCA Competition Policies](wca{documents}).
+4. Be publicly accessible.
+5. Be open to all competitors and spectators who wish to attend, up to the maximum capacity stated on the website or otherwise permitted by the venue.
+6. Be held on a single date or on two or more consecutive days.
+7. Have at least one official event.
+8. Have at least one competitor.
+   1. The competition should have at least 12 competitors.
+9. Have at least one designated WCA Delegate in attendance.
+   1. The WCA Delegate for a competition is responsible for:
+      1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
+      2. Sending the competition results to the WCA Results Team within one week of the end of the competition.
+         1. Results should be ready at the end of the last day of competition. Corrections to the results should be available within one week of the competition date.
+         2. All scramble sequences used during a competition must be sent with the results of the competition.
+            1. All scramble sequences must be labeled with the events, rounds, and groups for which they were used. If there are multiple groups on a round, it is not necessary to identify which competitor was in which group.
+         3. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days. Digital records (e.g. scramble sequences, results) must be retained permanently.
```

Comment:
Should the part about keeping digital records permanently be amended? This will likely be required after a future Data Protection Policy is in force, so we might as well take care of it now!

Reply:
I agree, it makes sense to change this now.

Reply:
@ChrisWrightWCA what do you think would be reasonable here?

Reply:
```suggestion
         3. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days. Digital records (e.g. scramble sequences, results) must be retained for 7 years.
```
Following offline discussion.

Reply:
The copy the Delegates' keeps yes, but of course not the actual data on our database :P Do you think this needs a clarification?

Reply:
yes